### PR TITLE
Rename Services to Actions in integrations: S

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

SSIA, adjust all occurrences for integrations starting with the letter S

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Updated terminology in integration documentation from "services" to "actions" for enhanced clarity and consistency.
  - Revised descriptions and examples to reflect the new terminology in the `switcher_kis`, `synology_chat`, `syslog`, `system_bridge`, and `system_log` integrations.
  - Updated a link in the `systemmonitor` integration documentation for improved accuracy.

### CODE REVIEW

**Review by joostlek — DISMISSED** [review]

**Review by bieniu — DISMISSED** [review]
Looks good

**Review by thecode — DISMISSED** [review]

File: source/_integrations/scene.markdown [hunk 1]
```diff
@@ -23,17 +23,17 @@ The scene entity is stateless, as in, it cannot have a state like the `on` or
...
+Some integrations like [Philips Hue](/integrations/hue), [MQTT](/integrations/mqtt), and [KNX](/integrations/knx) provide scenes. You can activate them from the Home Assistant UI or via as an action. In this case, the integration provides the preferred states to restore.
 
 ## Creating a scene
 
 You can create scenes that capture the states you want certain entities to be. For example, a scene can specify that light A should be turned on and light B should be bright red.
 
-Scenes can be created and managed via the user interface using the [Scene Editor](/docs/scene/editor/). They can also be manually configured via {% term "`configuration.yaml`" %}. Note that the entity data is not service call parameters, it's a representation of the wanted state:
+Scenes can be created and managed via the user interface using the [Scene Editor](/docs/scene/editor/). They can also be manually configured via {% term "`configuration.yaml`" %}. Note that the entity data are not action parameters, it's a representation of the wanted state:
```
Comment (c0ffeeca7) [inline source/_integrations/scene.markdown]:
```suggestion
Scenes can be created and managed via the user interface using the [Scene Editor](/docs/scene/editor/). They can also be manually configured via {% term "`configuration.yaml`" %}. Note that the entity data is not an action parameter; it's a representation of the wanted state:
```

File: source/_integrations/scene.markdown [hunk 2]
```diff
@@ -23,17 +23,17 @@ The scene entity is stateless, as in, it cannot have a state like the `on` or
...
+Some integrations like [Philips Hue](/integrations/hue), [MQTT](/integrations/mqtt), and [KNX](/integrations/knx) provide scenes. You can activate them from the Home Assistant UI or via as an action. In this case, the integration provides the preferred states to restore.
 
 ## Creating a scene
 
 You can create scenes that capture the states you want certain entities to be. For example, a scene can specify that light A should be turned on and light B should be bright red.
 
-Scenes can be created and managed via the user interface using the [Scene Editor](/docs/scene/editor/). They can also be manually configured via {% term "`configuration.yaml`" %}. Note that the entity data is not service call parameters, it's a representation of the wanted state:
+Scenes can be created and managed via the user interface using the [Scene Editor](/docs/scene/editor/). They can also be manually configured via {% term "`configuration.yaml`" %}. Note that the entity data is not an action parameter; it's a representation of the wanted state:
```
Comment (c0ffeeca7) [inline source/_integrations/scene.markdown]:
```suggestion
Scenes can be created and managed via the user interface using the [Scene Editor](/docs/scene/editor/). They can also be manually configured via {% term "`configuration.yaml`" %}. Note that entity data is not an action parameter; it's a representation of the wanted state:
```

File: source/_integrations/screenlogic.markdown [hunk 3]
```diff
@@ -32,17 +32,17 @@ The Pentair ScreenLogic integration allows you to integrate your Pentair Intelli
 
 ## Options
 
-ScreenLogic options are set via **Settings** -> **Devices & Services** -> **Pentair ScreenLogic** -> **Options**.
+ScreenLogic options are set via **Settings** -> **Devices & services** -> **Pentair ScreenLogic** -> **Options**.
```
Comment (c0ffeeca7) [inline source/_integrations/screenlogic.markdown]:
```suggestion
ScreenLogic options are set via **Settings** > **Devices & services** > **Pentair ScreenLogic** > **Options**.
```

File: source/_integrations/seventeentrack.markdown [hunk 4]
```diff
@@ -73,14 +73,14 @@ content: >
...
 {% endraw %}
 
-## Services
+## Actions
 
-### Service `seventeentrack.get_packages`
+### Action `seventeentrack.get_packages`
 
-The `seventeentrack.get_packages` service allows you to query the 17track API for the latest package data.
+the `seventeentrack.get_packages` action allows you to query the 17track API for the latest package data.
```
Comment (c0ffeeca7) [inline source/_integrations/seventeentrack.markdown]:
```suggestion
The `seventeentrack.get_packages` action allows you to query the 17track API for the latest package data.
```

File: source/_integrations/somfy_mylink.markdown [hunk 5]
```diff
@@ -18,4 +18,4 @@ The `Somfy MyLink` integration is used as an interface to a compatible Somfy MyL
 
 {% include integrations/config_flow.md %}
 
-If any of your devices are reversed, they can be adjusted in **Settings** -> **Devices & Services** -> **Somfy MyLink** >> `Options`.
+If any of your devices are reversed, they can be adjusted in **Settings** -> **Devices & services** -> **Somfy MyLink** >> `Options`.
```
Comment (c0ffeeca7) [inline source/_integrations/somfy_mylink.markdown]:
```suggestion
If any of your devices are reversed, they can be adjusted in **Settings** > **Devices & services** > **Somfy MyLink** > **Options**.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @frenck :+1:

**Pull request merged**